frontend/src/components/vestigingscijfers/CitoScore.vue

```diff
@@ -34,18 +41,21 @@ export default {
   data () {
     return {
       'citoScoreData': null,
-      'citoScoreGem': null
+      'citoScoreGem': null,
+      'MISSING_DATA': false
     }
   },
   async mounted () {
     this.getData()
   },
   methods: {
     async getData () {
-      let score = await readPaginatedData(API_HOST + `/onderwijs/api/cito-score/?vestiging=${this.id}&jaar=2016`)
-      this.citoScoreData = score[0].cet_gem
-      let scoreGem = await getBbgaVariables(['OCITOSCH_GEM'], ['STAD'], [2016])
-      this.citoScoreGem = scoreGem[0].waarde
+      this.citoScoreData = _.get(await readPaginatedData(API_HOST + `/onderwijs/api/cito-score/?vestiging=${this.id}&jaar=2016`), '[0].cet_gem', null)
+      this.citoScoreGem = _.get(await getBbgaVariables(['OCITOSCH_GEM'], ['STAD'], [2016]), '[0].waarde', null)
+
+      if (!this.citoScoreData) {
+        this.MISSING_DATA = true
+      }
     }
```

Comment:
Is MISSING_DATA needed? Looks like just an alias for !cityScoreData

Reply:
There are three relevant states:
1) Data is present and downloaded -> show it.
2) APIs work, data is not available for this school -> notify user that we do not have the data.
3) There is some other problem in the APIs (slow, offline) -> show the elipsis.

One boolean cannot encode 3 states hence the current solution.

Reply:
Well, if it is a state then make it a state I would say

Reply:
we talked about this, have implemented as agreed